test/evergreen_develop.yml

```diff
@@ -173,10 +173,14 @@ buildvariants:
     smp_command: -j $(echo "`grep -c ^processor /proc/cpuinfo` * 2" | bc)
     cmake_generator: Ninja
     make_command: ninja
+  # As these tests run infrequently by the time they fail we may have had 
+  # a lot of prior commits that would also fail for the same test. This is acceptable 
+  # as our infrequent tests aren't tracking critical failures, but we don't want 
+  # to perform stepback as evergreen will run the same test on all of these older commits 
+  # and generate a lot of redundant BFGs.
+  stepback: false
```

Comment:
The [Evergreen docs](https://docs.devprod.prod.corp.mongodb.com/evergreen/Test-with-Evergreen/Configure-a-Project/Project-Configuration-Files/#stepback) indicates `stepback` can be set/unset "at the top-level, and for individual tasks (in the task definition or for the task within a specific build variant)". Setting it at the build variant level is not mentioned. Better double-check with the Evergreen team on that.

Reply:
Confirmed with the team over Slack. They'll be updating the docs to explicitly mention that this flag can be set on buildvariants.